### PR TITLE
split_controller: refine the LOAD_BASE_SPLIT_EVENT metrics label definitions

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What is changed and how it works?

What's Changed:

Refine the `LOAD_BASE_SPLIT_EVENT ` metrics label definitions to make them more clear to help diagnose.

```commit-message
split_controller: refine the LOAD_BASE_SPLIT_EVENT metrics label definitions
```
### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

image

### Release note 

```release-note
None.
```

### CODE REVIEW

File: components/raftstore/src/store/worker/split_controller.rs [hunk 1]
```diff
@@ -22,6 +22,22 @@ use crate::store::worker::{FlowStatistics, SplitConfig, SplitConfigManager};
...
+// Split info has not been collected yet, not ready to split.
+const NOT_READY_TO_SPLIT: &str = "not_ready_to_split";
+// The number of sampled keys does not meet the threshold.
+const NO_ENOUGH_SAMPLED_KEY: &str = "no_enough_sampled_key";
+// The number of sampled keys located on left and right does not meet the threshold.
+const NO_ENOUGH_LR_KEY: &str = "no_enough_lr_key";
+// The number of balanced keys does not meet the score.
+const NO_BALANCE_KEY: &str = "no_balance_key";
+// The number of contained keys does not meet the score.
+const NO_UNCROSS_KEY: &str = "no_uncross_key";
```
Comment (rleungx) [inline components/raftstore/src/store/worker/split_controller.rs:39]: `uncross` is a little bit confused. But I don't have a better idea.

**Review by rleungx — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

**Review by disksing — APPROVED** [review]

**Pull request merged**